projects/cdk/directives/auto-focus/handlers/abstract.handler.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, ElementRef} from '@angular/core';
+import {
+    TuiFocusableElementAccessor,
+    TuiNativeFocusableElement,
+} from '@taiga-ui/cdk/interfaces';
+
+import {TuiAutofocusHandler} from '../autofocus.options';
+
+@Directive()
```

Comment:
It's `@Injectable`, not a directive.

Reply:
<img width="874" alt="image" src="https://user-images.githubusercontent.com/12021443/169296924-fc034de2-0107-48d0-a04a-4074ab1477ca.png">